cvat-ui/src/components/tasks-page/task-item.tsx

```diff
@@ -116,16 +116,16 @@ class TaskItemComponent extends React.PureComponent<TaskItemProps & RouteCompone
         const updated = moment(taskInstance.updatedDate).fromNow();
         const created = moment(taskInstance.createdDate).format('MMMM Do YYYY');
 
-        // Get and truncate a task name
-        const name = `${taskInstance.name.substring(0, 70)}${taskInstance.name.length > 70 ? '...' : ''}`;
-
         return (
             <Col span={10} className='cvat-task-item-description'>
-                <Text strong type='secondary' className='cvat-item-task-id'>{`#${id}: `}</Text>
-                <Text strong className='cvat-item-task-name'>
-                    {name}
-                </Text>
-                <br />
+                <Paragraph
+                    ellipsis={{ rows: 2, tooltip: taskInstance.name }}
+                >
+                    <Text strong type='secondary' className='cvat-item-task-id'>{`#${id}: `}</Text>
+                    <Text strong className='cvat-item-task-name'>
+                        {taskInstance.name}
+                    </Text>
+                </Paragraph>
```

Comment:
Perhaps it should be <Text>, not <Paragraph>.

Like:

```
<Text strong type='secondary' className='cvat-item-task-id'>{`#${id}: `}</Text>
<Text ellipsis={{ rows: 1, tooltip: taskInstance.name }} strong className='cvat-item-task-name'>
    {taskInstance.name}
</Text>
```

Because now we have only one line for a name. And in this case we do not need chaaning styles above.

Reply:
> That should be good enough.
> 
> ```
> <>
>     <Text strong type='secondary' className='cvat-item-task-id'>{`#${id}: `}</Text>
>     <Text ellipsis={{ tooltip: taskInstance.name }} strong className='cvat-item-task-name'>
>         {taskInstance.name}
>     </Text>
> </>
> ```
> 
> General recommendation is to be consistent here. If in three of four cases we do not show tooltip on hovering ID, then probably in the fourth case we should behave the same.

In one of two cases we are showing the tooltip upon hovering on id. The other cards dont have id in them just the cloud storage and task card have id.

We are choosing to not show the tooltip upon hovering on id?

Reply:
Fair enough. Let me left this decision up to you, then.